web/src/engine/websites/Kakaopage.ts

```diff
@@ -146,15 +141,19 @@ export default class extends DecoratableMangaScraper {
             productId: parseInt(chapter.Identifier),
             seriesId: parseInt(chapter.Parent.Identifier)
         };
-        const request = new Request(new URL('/graphql', this.URI).href, {
+
+        const { viewerInfo: { viewerData: { imageDownloadData: { files } } } } = await FetchGraphQL<APiPages>(this.CreateRequest(), 'viewerInfo', gql, vars);
+        return files.map(page => new Page(this, chapter, new URL(page.secureUrl)));
+
+    }
+
+    private CreateRequest(): Request {
+        return new Request(new URL('/graphql', this.apiUrl), {
             headers: {
```

Comment:
Headers seems redundant, aren't the defaults working?

Reply:
Nope.

![image](https://github.com/user-attachments/assets/9b8e234d-02b5-4471-87f5-6d970465feeb)